presto-docs/src/main/sphinx/sql/show-functions.rst

```diff
@@ -7,9 +7,10 @@ Synopsis
 
 .. code-block:: none
 
-    SHOW FUNCTIONS
+    SHOW FUNCTIONS [ LIKE pattern ]
```

Comment:
Also add escape?

Reply:
Done. Also added escape for `SHOW TABLE`.